charts/prometheus-postgres-exporter/templates/deployment.yaml

```diff
@@ -152,8 +152,10 @@ spec:
 {{- with .Values.extraContainers }}
 {{ toYaml . | indent 8 }}
 {{- end }}
-      securityContext:
-{{ toYaml .Values.securityContext | indent 8 }}
+          {{- with .Values.securityContext }}
```

Comment:
fyi, this still has a bug because of where this is placed, below extraContainers -- the change would be applied to the last container if sidecar containers are supplied, rather than the application container

Reply:
Apologies, you are of course right, thank you.